src/dialects/abstract/connection-manager.ts

```diff
@@ -189,8 +189,8 @@ export class AbstractConnectionManager<TConnection extends Connection = Connecti
     }
   }
 
-  async #initDatabaseVersion() {
-    if (this.sequelize.options.databaseVersion !== 0) {
+  async _initDatabaseVersion() {
+    if (this.sequelize.options.databaseVersion != null) {
```

Comment:
ok so `this._connect` doesn't exist in sqlite. We can try a different approach: Instead of this method calling `this._connect`, this method receives the connection to use

Reply:
ah got it!

Reply:
@ephys there is also an `_disconnect` tbh we don't really drop the connection we just provided for `sqlite` weird workaround would be an empty function for `_disconnect` or maybe we can just return? Because the `#versionPromise` uses `_disconnect` and that's something we don't wanna do for `sqlite`!

Reply:
sqlite does not fit well in our structure because it's a single-connection that always remains open and can be used concurrently, but it works